compiler/noirc_evaluator/src/ssa/opt/unrolling.rs

```diff
@@ -395,14 +822,19 @@ impl<'f> LoopIteration<'f> {
                 then_destination,
                 else_destination,
                 call_stack,
-            } => self.handle_jmpif(
-                *condition,
-                *then_destination,
-                *else_destination,
-                call_stack.clone(),
-            ),
+            } =>
+            // Check where the loop body wants to jump (e.g if-then blocks).
+            {
```

Comment:
nit: why this extra block `{ .. }`? For the comment? I think code states pretty well that we want to handle a if-then block

Reply:
You're right, it's probably not needed, I'll remove it (I added it to the method docstrings anyway). At the time it wasn't clear to me under which condition the blocks will have multiple destinations because I was only thinking about the loops, and wanted to capture that other kinds of blocks in the loop can still occur.